docs/sections/user_guide/yaml/components/make_hgrid.rst

```diff
@@ -0,0 +1,240 @@
+.. _make_hgrid_yaml:
+
+make_hgrid
+==========
+
+Structured YAML to run :ufs-utils:`make_hgrid<make-hgrid>` is validated by JSON Schema and requires the ``make_hgrid:`` block, described below. If ``make_hgrid`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+YAML keys and their descriptions are taken largely verbatim from the ``make_hgrid`` tool source code, available `here <https://github.com/NOAA-GFDL/FRE-NCtools/blob/65c34e76cfa2ff567f70da83f44cd6f991315f08/tools/make_hgrid/make_hgrid.c#L71>`_.
+
+Here is a prototype UW YAML ``make_hgrid:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/make_hgrid.yaml
+
+UW YAML for the ``make_hgrid:`` Block
+-------------------------------------
+
+execution:
+^^^^^^^^^^
+
+See :ref:`here <execution_yaml>` for details.
+
+run_dir:
+^^^^^^^^
+
+The path to the directory where ``make_hgrid`` will write its outputs.
+
+config:
+^^^^^^^
+
+Describes the required parameters to run a ``make_hgrid`` configuration. 
+
+
+grid_type:
+""""""""""
+
+Specify grid type with one of the following recognized key words:
+
+.. list-table:: Grid Types and Requirements
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Grid Type
+     - Requirements
+   * - from_file
+     - --my_grid_file must be specified.
```

Comment:
```suggestion
   * - ``from_file``
     - ``my_grid_file:`` must be specified.
```
I think the `--` prefixes here are confusing, as they refer to CLI switches that UW users are not directly exposed to. I think the suggested formatting would help the user connect this advice to the `my_grid_file:` section in the docs below.

Same suggestion applies to all the `--` CLI switches mentioned in this table.

Any reason to also not sort these?

Reply:
I let both of these slip through when fighting with the table format, I'll clear this up

Reply:
Fixed it, it should be ok here